Content.Shared/Rotation/RotationVisualsComponent.cs

```diff
@@ -9,14 +9,16 @@ public sealed partial class RotationVisualsComponent : Component
     /// <summary>
     /// Default value of <see cref="HorizontalRotation"/>
     /// </summary>
+    ///
+    //should we not be changing the default rotation to be either horizontal or vertical?
     [DataField]
-    public Angle DefaultRotation = Angle.FromDegrees(90);
+    public Angle DefaultRotation = Angle.Zero; //should this not be 0 degrees? as the character stands up by default (as opposed to 90 degrees - laying down)
```

Comment:
AFAIK the variable is just badly named, its meant to be the default rotation an entity should have when horizontal. I think it just came about as a result of some buckling shitcode to stop dead people from being horizontal while buckled to a chair. So AFAIK it is meant to be 90.
@AC